### PR TITLE
Issue 638 p2

### PR DESCRIPTION
**Pull request by 6za**
- Re-enable the use of PAT for github via OS var

### CODE REVIEW

File: cmd/init.go [hunk 1]
```diff
@@ -77,10 +79,17 @@ validated and configured.`,
...
+				log.Println("Error trying to capture github token")
+			}
+			if githubToken != "" {
+				//viper.Set("github.token", os.Getenv("GITHUB_AUTH_TOKEN"))
+				log.Println("Github token found")
+				// Do nothing, we dont set to viper anymore
 			} else {
-				log.Fatal("cannot create a cluster without a github auth token. please export your GITHUB_AUTH_TOKEN in your terminal.")
+				log.Println("cannot create a cluster without a github auth token. please export your GITHUB_AUTH_TOKEN in your terminal.")
+				return fmt.Errorf("missing github token, unable to continue")
```
Comment (6za) [inline cmd/init.go:92]: This seems a bit ugly, but I think it should work, I will test on my next test.

**Comment by converge** [comment]
I'll run some local command tests to make sure it's not breaking the local command

**Comment by converge** [comment]
there is something to adjust:

```
# github.com/kubefirst/kubefirst/internal/gitClient
internal/gitClient/git.go:339:10: too many return values
	have (error)
	want ()
```

**Comment by 6za** [comment]
> there is something to adjust:
> 
> ```
> # github.com/kubefirst/kubefirst/internal/gitClient
> internal/gitClient/git.go:339:10: too many return values
> 	have (error)
> 	want ()
> ```

just fixed

**Comment by converge** [comment]
I just finished a full local run with success, no side-effects 👍 

question, both local command viper and flagset implementation uses prefix for env. variables names, should we make it standard?

like, instead of GITHUB_AUTH_TOKEN, KUBEFIRST_GITHUB_AUTH_TOKEN

**Comment by converge** [comment]
there is something impacting the `destroy` command, Im on it

**Review by marinhojess — APPROVED** [review]

**Comment by 6za** [comment]
This is been handled on another PR.

**Pull request closed without merging**